subprojects/docs/src/docs/userguide/installation.adoc

```diff
@@ -15,69 +15,83 @@
 [[installation]]
 = Installing Gradle
 
-You can install the Gradle build tool on Linux, macOS, or Windows.
-This document covers installing using a package manager like SDKMAN! or Homebrew, as well as manual installation.
+You can install Gradle Build Tool on Linux, macOS, or Windows.
 
-Use of the <<gradle_wrapper.adoc#sec:upgrading_wrapper,Gradle Wrapper>> is the recommended way to upgrade Gradle.
+The installation can be done manually or using a package manager like https://sdkman.io/[SDKMAN!] or https://brew.sh/[Homebrew].
 
-You can find all releases and their checksums on the link:{website}/releases[releases page].
+Please use the <<gradle_wrapper.adoc#sec:upgrading_wrapper,Gradle Wrapper>> to upgrade Gradle.
```

Comment:
```suggestion
Please use the <<gradle_wrapper.adoc#sec:upgrading_wrapper,Gradle Wrapper>> to upgrade Gradle.
```
I've been wanting to make this change:
```suggestion
Use the <<gradle_wrapper.adoc#sec:upgrading_wrapper,Gradle Wrapper>> to upgrade Gradle.
```

Reply:
@lkasso Thanks, but this PR was not supposed to change the docs, and this diff came from merging `master` into the branch. It should be gone now.